credentials/src/main/kotlin/SSI.kt

```diff
@@ -105,11 +113,9 @@ public data class CreateVcOptions(
 )
 
 public data class CreateVpOptions(
-  // TODO: Add this
-  // val presentationDefinition: PresentationDefinitionV2,
-  // val verifiableCredentialJwts: List<String>,
-  // TODO: remove this for verifiableCredentialJwts instead
-  val verifiableCredentials: List<VerifiableCredentialType>,
+  val presentationDefinition: PresentationDefinitionV2,
```

Comment:
Since this is part of the public API, would it make sense to start adding kotlin docs for all fields?